### PR TITLE
[P1-L14] Add events to store state fee update

### PR DESCRIPTION
**Pull request by chrismaree**
Right now the store does not emit events after fees being updated. This PR adds them and associated tests.

Open question: I dont really see enough test coverage for weekly delay fees. This should be expanded.

### CODE REVIEW

**Comment by mrice32** [comment]
> Open question: I dont really see enough test coverage for weekly delay fees. This should be expanded.

I think there's a separate issue in the audit on this exact point.

File: core/test/oracle/Store.js [hunk 1]
```diff
@@ -64,13 +65,30 @@ contract("Store", function(accounts) {
     // TODO Check that only permitted role can change the fee
   });
 
-  it("Final fees", async function() {
+  it.only("Final fees", async function() {
```
Comment (mrice32) [inline core/test/oracle/Store.js:68]:
```suggestion
  it("Final fees", async function() {
```

File: core/test/oracle/Store.js [hunk 2]
```diff
@@ -64,13 +65,30 @@ contract("Store", function(accounts) {
...
+    );
+
+    truffleAssert.eventEmitted(result, "NewFinalFee", ev => {
+      return ev.newFinalFee.rawValue === web3.utils.toWei("5", "ether");
+    });
     const fee = await store.computeFinalFee(arbitraryTokenAddr);
     assert.equal(fee.rawValue, web3.utils.toWei("5", "ether"));
   });
 
+  it.only("Weekly delay fees", async function() {
```
Comment (mrice32) [inline core/test/oracle/Store.js:83]:
```suggestion
  it("Weekly delay fees", async function() {
```

File: core/test/oracle/Store.js (hunk 1 above)
Comment (chrismaree) [reply core/test/oracle/Store.js]: this was an oversight, sorry!

File: core/test/oracle/Store.js (hunk 2 above)
Comment (chrismaree) [reply core/test/oracle/Store.js]: this was a mistake, sorry! submitted the PR too quickly.

**Pull request merged**